### PR TITLE
Fix bcmf_netdev.c:705:7: error: 'strnlen' specified bound 2 exceeds source size 1 [-Werror=stringop-overread]

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
Report here: https://github.com/apache/incubator-nuttx-apps/actions/runs/3303013192/jobs/5450504682

## Impact
Minor

## Testing
Pass CI

### CODE REVIEW

File: drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c [hunk 1]
```diff
@@ -702,7 +702,7 @@ static int bcmf_ifup(FAR struct net_driver_s *dev)
       goto errout_in_wl_active;
     }
 
-  if (strnlen(CONFIG_IEEE80211_BROADCOM_DEFAULT_COUNTRY, 2) == 2)
+  if (CONFIG_IEEE80211_BROADCOM_DEFAULT_COUNTRY[0])
```
[the remark this answers is not in the document]
Comment (acassis) [reply drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c:705]:
```suggestion
  if (CONFIG_IEEE80211_BROADCOM_DEFAULT_COUNTRY != NULL &&
      CONFIG_IEEE80211_BROADCOM_DEFAULT_COUNTRY[0])

[the remark this answers is not in the document]
Comment (xiaoxiang781216) [reply drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c:705]: You can't, CONFIG_IEEE80211_BROADCOM_DEFAULT_COUNTRY come from Kconfig, the minimum value is "".

[the remark this answers is not in the document]
Comment (acassis) [reply drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c:705]: Right, but protection is always welcome! ;-)

[the remark this answers is not in the document]
Comment (xiaoxiang781216) [reply drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c:705]: But it's a compile time constant, compiler can report the abnormal usage.

[the remark this answers is not in the document]
Comment (pkarashchenko) [reply drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c:705]: I think we do not need check for constant as it will be caught at compile time.

**Review by pkarashchenko — APPROVED** [review]

**Comment by pkarashchenko** [comment]
Merging this to resolve dependency on apps repo

**Pull request merged**